vip-application-importer/src/main/java/fr/insalyon/creatis/vip/applicationimporter/client/view/BoutiquesImportLayout.java

```diff
@@ -144,6 +144,10 @@ public void onSuccess(String jsonFileContent) {
                                 Constants.TAB_ID_BOUTIQUES_APPLICATION,
                                 Constants.TAB_NAME_BOUTIQUES));
                     displayTab.setBoutiqueTool(boutiquesTool);
+                    if (boutiquesTool.setWarning() != null) {
```

Comment:
The `DisplayTab.parseJSON` method called line 138 de a boutiques verification, it is better to make the command-line verification there.

Reply:
Moved the varification to DisplayTab.parseJSON method